### PR TITLE
Expand hotspot and enable hover on hotspot in static scene

### PR DESCRIPTION
**Pull request by EivinS**

### CODE REVIEW

File: scripts/components/Interactions/HotspotNavButton.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default class HotspotNavButton extends React.Component {
     we have to multiply the result by two*/
 
     let finalValue = ((currentPosMouse - divStartWidth) * 2);
-    if(finalValue > 128 && finalValue < 512) {
+    if(finalValue > 128 && finalValue < 2048) {
```
Comment (boyum) [inline scripts/components/Interactions/HotspotNavButton.js:82]: Does this affect height as well?

Comment (EivinS) [reply scripts/components/Interactions/HotspotNavButton.js:82]: yes

Comment (EivinS) [reply scripts/components/Interactions/HotspotNavButton.js:82]: divStartWidth may not be the correct word though

Comment (EivinS) [reply scripts/components/Interactions/HotspotNavButton.js:82]: Changed it to divStartPos

**Review by boyum — APPROVED** [review]

**Pull request merged**